app/src/main/java/com/evelynhasama/bucketbuds/ActivityDetailsFragment.java

```diff
@@ -395,9 +444,59 @@ else if (item.getItemId() == MenuHelper.CALENDAR){
             afterSetStart = false;
             showDatePickerDialog(view, true);
         }
+        else if (item.getItemId() == MenuHelper.MAP){
+            Uri gmmIntentUri = Uri.parse(activityObj.getAddress());
+            Intent mapIntent = new Intent(Intent.ACTION_VIEW, gmmIntentUri);
+            mapIntent.setPackage("com.google.android.apps.maps");
+            startActivity(mapIntent);
+        }
         return super.onOptionsItemSelected(item);
     }
 
+    private void getLocation() {
 
+        LocationManager locationManager = (LocationManager) getActivity().getSystemService(Context.LOCATION_SERVICE);
+        if (locationManager != null) {
+            // check permissions
+            if (ActivityCompat.checkSelfPermission(getContext(), Manifest.permission.ACCESS_FINE_LOCATION) != PackageManager.PERMISSION_GRANTED && ActivityCompat.checkSelfPermission(getContext(), Manifest.permission.ACCESS_COARSE_LOCATION) != PackageManager.PERMISSION_GRANTED) {
+                ActivityCompat.requestPermissions(getActivity(), new String[]{Manifest.permission.ACCESS_FINE_LOCATION, Manifest.permission.ACCESS_COARSE_LOCATION}, LOCATION_REQUEST_PERMISSION_CODE);
+                return;
+            }
+            // getting GPS status
+            if (!locationManager.isProviderEnabled(LocationManager.GPS_PROVIDER)){
+                Toast.makeText(getContext(), "Enable GPS", Toast.LENGTH_SHORT).show();
+                return;
+            };
+            Location location = locationManager.getLastKnownLocation(LocationManager.GPS_PROVIDER);
+            Double latitude;
+            Double longitude;
+            if (location != null) {
+                latitude = location.getLatitude();
+                longitude = location.getLongitude();
+                LatLng northEast = new LatLng(latitude + 0.2, longitude + 0.2);
```

Comment:
nit: Would be nice to have `0.2` declared as a constant with a comment explaining more or less what it means, even if it was chosen arbitrarily.